### DISCUSSION TITLE
How to directly overwrite the source file with the new file after the rotation？

### DISCUSSION THREAD

**Discussion by tyf2018** [qpdf/qpdf#784] [Q&A] [answered]
I use the below command to rotate the PDF page and it works fine.
`A.pdf B.pdf --rotate=+90:1` 

Now I want to directly overwrite the source file with the new file after the rotation, but the command is modified as follows and only get a blank pdf file.
`A.pdf A.pdf --rotate=+90:1` 
What should I do to get the desired result?

Best regards.

**Comment by jberkenbilt** [comment, marked as answer]
```
qpdf --replace-input A.pdf --rotate=+90:1
```
I'm surprised you get a blank file though. When I run this, I get

```
qpdf: input file and output file are the same; use --replace-input to intentionally overwrite the input file
```
The `--replace-input` option was added in qpdf 9.0.0. Are you running an older version?

**Reply by tyf2018** [reply]
Thanks. 

> Are you running an older version?

The version of qpdf is 11.1. I am using this command in Total Commander.
